tool/dependencies/known-dependencies.txt

```diff
@@ -589,6 +614,7 @@ xmlbeans-5.1.1.jar
 xmlenc-0.52.jar
 xstream-1.4.20.jar
 xz-1.5.jar
+zjsonpatch-0.3.0.jar
 zookeeper-3.5.9.jar
 zookeeper-jute-3.5.9.jar
```

Comment:
need update  license file  at  `linkis-dist/release-docs/licenses`  for new dependencies 
can refer to https://linkis.apache.org/zh-CN/docs/latest/development/development-specification/license#21-%E7%A4%BA%E4%BE%8B-%E5%9C%BA%E6%99%AF1

Reply:
Ok, thanks for reviewing the code